nixos/modules/services/system/dbus.nix

```diff
@@ -16,11 +14,11 @@ let
     serviceDirectories = cfg.packages;
   };
 
+  inherit (lib) mkOption types;
```

Comment:
IMO not a fan of maintaining such inherits

Reply:
I think it's better than a global `with` or dotting `lib.` everywhere. It's a bit odd but I think it's the best option out of the bunch.

Reply:
I have had this same conversation with @SuperSandro2000 and I"m with @Atemu on this.